src/View/Widget/DateTimeWidget.php

```diff
@@ -91,6 +91,9 @@ class DateTimeWidget extends BasicWidget
      * - `escape` Set to false to disable escaping on all attributes.
      * - `type` A valid HTML date/time input type. Defaults to "datetime-local".
      * - `timezone` The timezone the input value should be converted to.
+     * - `step` The stepping interval to use for this input (seconds). Default is based upon given type.
+     * - `format` The format of the time/date input. Default is based upon given type.
+     * - `second` Set to false to disable seconds and remove default step when using "datetime-local" or "time" type.
```

Comment:
I don't see the need for a `second` option. Hiding seconds value can be achieved using setting required `format` only.

Reply:
We can certainly achieve the same result simply by setting `step` to `null` and `format` to something like `'H:i'`, and I'm not against this simplified solution, but there was some precedence in [version 3.x](https://book.cakephp.org/3/en/views/helpers/form.html#options-for-time-related-controls) that I thought might be beneficial for those migrating to 4.x.

Reply:
We wanted to make the code simpler which is why we got rid of the multiple selects for datetime and related options of 3.x :slightly_smiling_face:.

I have opened an alternate PR #14614.

Reply:
I understand -- PR looks good to me, thanks!